tests/terraform/graph/graph_builder/test_oci_policy.py

```diff
@@ -1,43 +1,38 @@
+"""Test if OCI policy statements are evaluated correctly."""
+
 import os
-import unittest
+from pathlib import Path
+from unittest import mock
 
-from checkov.common.graph.db_connectors.networkx.networkx_db_connector import (
-    NetworkxConnector,
-)
+from checkov.common.graph.db_connectors.networkx.networkx_db_connector import NetworkxConnector
 from checkov.terraform.graph_manager import TerraformGraphManager
 
+@mock.patch.dict(os.environ, {"CKV_TERRAFORM_PROVIDER": "OCI"})
```

Comment:
I also considered mocking the `env` variable but because `attributes_no_eval` is extended at module load time, the test wouldn't have exercised my change exactly. So I decided to set the `env` variable before running `pytest` in the GHA workflow to make sure it doesn't have any side effects. If you think mocking is enough for testing, I'm fine with it.

Reply:
yeah, it only works because I changed the global variable to be an instance variable, otherwise this approach wouldn't work, because they are only loaded once.